templates/taxi/car_list.html

```diff
@@ -1,16 +1,38 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list<a style="float: right" href="{% url 'taxi:car-create' %}">+</a></h1>
+
   {% if car_list %}
-    <ul>
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Model</th>
+        <th>Manufacturer</th>
+        <th>Update</th>
+        <th>Delete</th>
+      </tr>
+
       {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
+        <tr>
+          <td>
+            <a href="{% url 'taxi:car-detail' pk=car.id %}">{{ car.id }}</a>
+          </td>
+          <td>
+            {{ car.model }}
+          </td>
+          <td>
+            {{ car.manufacturer.name }}
+          </td>
+          <td>
+            <a href="{% url 'taxi:car-update' pk=car.id %}">UPDATE</a>
+          </td>
+          <td>
+            <a href="{% url 'taxi:car-delete' pk=car.id %}">DELETE</a>
```

Comment:
Code Quality Issue: It's recommended to have a 'Cancel' button along with the 'Delete' button. The 'Cancel' button will lead to the previous page the user was on. It helps to prevent accidental deletions.